cluster-autoscaler/SYNC-CHANGES/SYNC_CHANGES-1.34.md

```diff
@@ -0,0 +1,36 @@
+<!--- For help refer to https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.20.md?plain=1 as example --->
+
+- [v1.34.0](#v1340)
+    - [Synced with which upstream CA](#synced-with-which-upstream-ca)
+    - [Changes made](#changes-made)
+        - [Changes during go.mod update](#changes-during-gomod-update)
+        - [Others](#others)
+
+# v1.34.0
+
+## Synced with which upstream CA
+[v1.34.0](https://github.com/kubernetes/autoscaler/tree/cluster-autoscaler-1.34.0/cluster-autoscaler)
+
+## Changes made
+- See general release notes of 1.34.0: https://github.com/kubernetes/autoscaler/releases/tag/cluster-autoscaler-1.34.0
+- Binpacking simulator will now consider old nodes when trying to pack pods with topology spread contraints in order to avoid unnecessary scale ups.
+- Default value for `--cordon-node-before-terminating` changed to `True`.
+- The KeepPartiallyFailedZeroOrMaxScalingNodeGroups field has been added to NodeGroupAutoScalingOptions to not allow nodes from the node-groups to be deleted if any node has creation errors i.e. the node-group is left unmodified unless all their nodes fail.
```

Comment:
```suggestion
- New field `KeepPartiallyFailedZeroOrMaxScalingNodeGroups` added to `NodeGroupAutoScalingOptions` to prevent nodes from the node-groups being deleted if they have creation errors i.e. the node-group is left unmodified unless all their nodes fail.
```